plugins/inputs/gnmi/sample.conf

```diff
@@ -13,6 +13,26 @@
   ## redial in case of failures after
   # redial = "10s"
 
+  ## gRPC Keepalive settings
+  ## See https://pkg.go.dev/google.golang.org/grpc/keepalive
+  ## After a duration of this time if the client doesn't see any activity it
+  ## pings the server to see if the transport is still alive.
+  ## If set and set below 10s, a minimum value of 10s will be used instead.
+  ## If not set, none of the other keepalive values will be configured.
+  # keepalive_time = "10s"
+
+  ## After having pinged for keepalive check, the client waits for a duration
+  ## of Timeout and if no activity is seen even after that the connection is
+  ## closed.
+  ## The current default value in GRPC is 20 seconds.
+  ## keepalive_timeout = "20s"
+
+  ## If true, client sends keepalive pings even with no active RPCs. If false,
+  ## when there are no active RPCs, Time and Timeout will be ignored and no
+  ## keepalive pings will be sent.
+  ## false by default in GRPC
+  # keepalive_permit_without_stream = false
```

Comment:
Do we really have this case? We always will establish a RPC, won't we?

Reply:
I just plumbed through what was there!

Reply:
To expand on that a little bit, since all of the settings are ultimately configured in one struct, I thought it simplest to just plumb all of them through, and leave it future proofed.  I see your point that there will always be subscriptions, so maybe it makes sense to remove it and just leave the default value in the struct (since it doesn't ultimately matter).

Reply:
Yeah I would prefer that. We can still add the option back later if we need it. My fear is to cause confusion on when this is effective...

Reply:
Removed!